plugins/rikki/loungeviews/components/Navigation.php

```diff
@@ -38,6 +39,10 @@ public function componentDetails()
     public $past_amateurseasons = null;
     public $divsseasons = null;
 
+
+    public $hasMatches = false;
+    public $seasons = null;
+
```

Comment:
Are these required?

Reply:
Missed these when I copied over the old sidenav info. they didn't appear to be needed, so removed them.